smtp/init.lua

```diff
@@ -151,10 +151,10 @@ curl_mt = {
         --
         request = function(self, url, from, to, body, opts)
             opts = opts or {}
-            to = to or {}
-            if not body or not url or not from then
+            if not body or not url or not from or not (to or opts.cc or opts.bcc) then
```

Comment:
So `nil` and `{}` for `to` will be interpreted differently. That's strange.

Reply:
Fixed, added tests for both nil, empty string and empty table cases